### DISCUSSION TITLE
Trying to create a manual

### DISCUSSION THREAD

**Discussion by raydeleu** [zalo/CascadeStudio#96] [Show and tell]
Over the last days I have experimented a lot with CascadeStudio. I have a lot of experience in other 3D modelling tools (Cinema4D, Blender, CSI Concepts (now SharkCad), FreeCAD, Moi3D/Rhino3D). Recently I started to model some shapes in OpenSCAD and through that I found Cascade Studio. Although I am quite used to find my way by trial and error I missed a simple manual for Cascade Studio. That is why I started to write one, using information from the javascript library and the Intellisense file as a starting point. I intend to add to this document as I experiment with more and more functions. 

Please feel free to provide corrections, amendments or additional material. 

https://github.com/raydeleu/CascadeStudioManual/blob/main/readme.adoc

**Comment by Irev-Dev** [comment]
Thanks for putting this together. I'm sure it was no small amount of work @raydeleu.

For reference the only other docs I know of for CascadeStudio is this tutorial on medium. https://ronie.medium.com/cascade-studio-tutorial-ee2f1c42c829

Obviously Github understand the `.adoc` extension since it rendering your manual well. If you change the name of the file to readme.adoc would that make the docs show up when viewing the repo instead of having to click through to CCS_Manual_main.adoc?

**Reply by raydeleu** [reply]
I changed the name to readme.adoc and it seems that it indeed shows up without a problem. Thanks for the tip, I am not that familiar with github and just a hobby-coder :-)

**Comment by raydeleu** [comment]
I have completed all chapters of the manual. The only part missing is a quick reference chart, but this could also be published separately. Please feel free to send me comments, corrections and other ideas to make the manual better.

**Comment by johnsonm** [comment]
I just discovered this work of art. Thanks @raydeleu — it's not just a manual, it's an education. ☺
